lib/pbench/test/unit/server/test_datasets_inventory.py

```diff
@@ -65,7 +66,7 @@ def test_dataset_not_present(self, query_get_as):
     def test_unauthorized_access(self, query_get_as):
         response = query_get_as("test", "metadata.log", HTTPStatus.FORBIDDEN)
         assert response.json == {
-            "message": "User drb is not authorized to READ a resource owned by test with private access"
+            "message": f"User drb is not authorized to READ a resource owned by {TEST_USER_ID} with private access"
```

Comment:
I'm struck here by the inconsistency of reporting a *username* for the authenticated user but a user *ID* for the dataset. This is painful. I'm reluctantly accepting that this is an ugly intermediate phase before things come back together and we have reverse-translate through the new user table, but it's still ugly. Sigh. 😦